models/topics.js

```diff
@@ -0,0 +1,15 @@
+const db = require('../db/connection.js');
+
+exports.selectTopics = () => {
+    return db 
+    .query('SELECT * FROM topics;')
+}
+
+exports.selectArticles = () => {
+    return db 
+    .query(`SELECT articles.*, (comments.article_id) comment_count
+            FROM articles
+            JOIN users ON users.username = articles.author
+            JOIN comments ON comments.article_id = articles.article_id
```

Comment:
Consider using another type of join here as currently articles without comments are not included.